src/main/java/Deadline.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Deadline Class.
+ * Deadline Class is a subclass of Task Class. It stores the description of this instance
+ * as well as the date and time.
+ *
+ * @author Amos Cheong
+ */
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    /**
+     * Get the description of this Deadline object.
+     * @return String description of this Deadline object.
+     */
+    public String getDesc() {
+        return super.description;
+    }
+
+    /**
+     * Get the date and time of this Deadline object.
```

Comment:
This is a small detail, but you can leave a blank line between the method description and the return type in the javadoc comment like you did for the getType() method.

Also, you don't need to write javadoc comments for self-explanatory methods like getDate() and getType().

This is a well written class. Good job!!